.github/workflows/krew-release.yml

```
@@ -0,0 +1,11 @@
+on:
+  release:
+    types: [released]
+jobs:
+  krew-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
```

```suggestion
        uses: actions/checkout@v4
```

Could you specify the released version so that we can avoid falling the rabbit hole?

---

Done